pkgs/tools/misc/natls/default.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
```

> Arguments should be listed in the order they are used, with the exception of lib, which always goes first

see https://nixos.org/manual/nixpkgs/unstable/#sec-syntax

```suggestion
, rustPlatform
, fetchFromGitHub
```